tensorflow/python/ops/gradients_impl.py

```diff
@@ -1010,26 +1008,23 @@ def hessians(ys, xs, name="hessians", colocate_gradients_with_ops=False,
   hessians = []
   _gradients = gradients(ys, xs, **kwargs)
   for i, _gradient, x in zip(range(len(xs)), _gradients, xs):
-    # Ensure that x is a vector.
-    check_rank = check_ops.assert_rank(
-      x, 1, message='Cannot compute Hessian because element %d of `xs` does '
-      'not have rank one.' % i
+    # tricky change shape to one-dimension without graph branching
```

Comment:
Is this a trick to reshape the tensor or is it "tricky" in the sense that it is difficult?

Reply:
That's means that It is not a general approach.
It would be better to erase that word. I will be fix it